app/index.html

```diff
@@ -58,80 +58,10 @@
-                                </div>
-                                <div class="tp-caption italic large_white_light sft hidden-xs" data-x="right" data-y="center" data-hoffset="0" data-voffset="0" data-speed="1000" data-start="1300" data-easing="Power4.easeOut">
-                                    <a href="#team" class="btn btn-naked">Meet The Team</a>
-                                </div>
-                            </li>
-                            <li data-transition="slotfade-horizontal" data-slotamount="7" data-masterspeed="500">
-                                <!-- MAIN IMAGE -->
-                                <img src="img/dummy.png" data-lazyload="img/samples/slider-bk2.jpg" alt="slidebg1" data-bgfit="cover" data-bgposition="center center" data-bgrepeat="no-repeat">
-                                <!-- LAYERS -->
-
-                                <!-- LAYER NR. 1 -->
-                                <div class="tp-caption lfl fadeout" data-x="left" data-y="center" data-hoffset="0" data-voffset="0" data-speed="500" data-start="700" data-easing="Power4.easeOut">
-                                    <img src="img/samples/iphone.png" alt="">
-                                </div>
-                                <div class="tp-caption large_white_bold sft" data-x="520" data-y="center" data-hoffset="0" data-voffset="-120" data-speed="500" data-start="1200" data-easing="Power4.easeOut">
-                                    <h1>.welcome</h1>
-                                </div>
-                                <div class="tp-caption large_white_light sfb" data-x="520" data-y="center" data-hoffset="0" data-voffset="-40" data-speed="1000" data-start="1300" data-easing="Power4.easeOut">
-                                    We Develop
-                                </div>
-                                <div class="tp-caption large_white_light sfb" data-x="520" data-y="center" data-hoffset="0" data-voffset="10" data-speed="1000" data-start="1300" data-easing="Power4.easeOut">
-                                    Beautiful & Functional
-                                </div>
-                                <div class="tp-caption large_white_light sfb" data-x="520" data-y="center" data-hoffset="0" data-voffset="60" data-speed="1000" data-start="1300" data-easing="Power4.easeOut">
-                                    Websites, Bradings & Web Apps
-                                </div>
-                                <div class="tp-caption sfb hidden-xs" data-x="520" data-y="center" data-hoffset="0" data-voffset="150" data-speed="1000" data-start="1700" data-easing="Power4.easeOut">
-                                    <a href="#about" class="btn btn-primary btn-lg">LEARN MORE</a>
-                                </div>
-                            </li>
-
-                        </ul>
-                        <div class="tp-bannertimer"></div>
-                    </div>
+                    <video autoplay style="width:100%">
```

Comment:
Sure I can do that. In production though, the video just be autoplaying in the background right?